schema/tests/architecture-dependent.rs

```diff
@@ -0,0 +1,24 @@
+use impls::impls;
+use iroha_schema::IntoSchema;
+use parity_scale_codec::{Decode, Encode};
+
+#[test]
+fn usize_isize_not_into_schema() {
+    // The architecture-dependent
+    assert!(!impls!(usize: IntoSchema));
+    assert!(!impls!(isize: IntoSchema));
```

Comment:
What about to use compile-time assertion?

```rust
/// Macro to abort compilation, if `e` isn't `true`
macro_rules! const_assert {
    ($e:expr) => {
        #[allow(trivial_casts)]
        const _: usize = ($e as bool) as usize - 1;
    };
}
```

This one I used in `client/test/integration/triggers/time_trigger.rs`